share/spack/qa/windows_test_setup.ps1

```diff
@@ -1,11 +1,4 @@
-Set-Location ../
-
-$env:python_pf_ver="C:\hostedtoolcache\windows\Python\3.9.5\x64\python.exe"
-
-cmd /c "`"spack\bin\spack_cmd.bat`" print " |
-foreach {
-    if ($_ -match "=") {
-        $v = $_.split("=")
-        [Environment]::SetEnvironmentVariable($v[0], $v[1])
-    }
-}
+$ErrorActionPreference = "SilentlyContinue"
+Write-Output F|xcopy .\share\spack\qa\configuration\windows_config.yaml $env:USERPROFILE\.spack\windows\config.yaml
+(Get-Item '.\lib\spack\docs\_spack_root').Delete()
```

Comment:
(question) I assume there is some issue with this being a symlink back into Spack is it just that it doesn't work on Windows? Or is the cyclic behavior causing problems? A comment might be useful.

Reply:
I don't think I actually added this line in this PR, but it doesn't seen to exist in develop. I truly can't remember adding it and I made this PR like a month ago, so it's still pretty fresh. When I check the blame locally it says I committed it back in January... IIRC this link causes havoc with the installer and a few other operations on Windows. The problem is the cyclic behavior, I don't think CMake handles it well at the moment. I'll add a comment.